hack/operator-sdk-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+make operatorSDK
+
+file_path="$(pwd)/bundle/manifests/addon-operator.clusterserviceversion.yaml"
+
+if [ -e "$file_path" ]; then
```

Comment:
- Double brackets don't spin up a separate process thread while the single ones do.
```suggestion
if [[ -e "$file_path" ]]; then
```